src/components/VMapbox.vue

```diff
@@ -178,7 +178,7 @@ export default {
     },
 
     resize() {
-      if (this.map) {
+      if (this.map && this.map.getCanvas()) {
```

Comment:
To prevent this error 
![image](https://github.com/openearth/vue2mapbox-gl/assets/15196342/2428f32e-a77d-4b5a-80fc-a535286aa349)